packages/transmute-cli/src/index.js

```diff
@@ -29,10 +30,21 @@ const mixpanel = process.env.MIXPANEL_PROJECT_ID
     @function
 */
 vorpal
-  .command('login')
-  .description(
-    'Login with okta, and save session to ~/.transmute/cli-secrets/session.json'
-  )
+  .command('test')
+  .description('Login to service')
+  .action(function(args, callback) {
+    runtest.test()
+    callback();
+  });
+
+vorpal
+  .command('login <loginService>')
```

Comment:
there is no login service, and none of these options are supported. please remove.